### ISSUE TITLE
Line jumps when the pointer is clicked near it

### ISSUE THREAD

**Issue by elkins0** [CindyJS/CindyJS#78]
When you click and release the pointer near the line thru a point the line jumps back and forth across the pointer each time you do this. Doing this near the point A does not do this.

The proposed fix shown below eliminates this behavior.

In https://github.com/CindyJS/CindyJS/tree/master/src/js/Events.js:
```javascript
function getmover(mouse) { ...
        } else if (el.kind === "L") { ...
```
Change lines 50 & 51 to negate the values of dx and dy:

```javascript
            dx = ln.value[0].value.real * dist;
            dy = ln.value[1].value.real * dist;
```

to:

```javascript
            dx = -ln.value[0].value.real * dist;
            dy = -ln.value[1].value.real * dist;
```

**Comment by gagern** [comment]
I can reproduce both your observation and your fix. Looks good to me. Can you create a pull request for this? Since the fix is so simple, I'd suggest you try using the GitHub web interface for this: simply click on the pen icon in the top right corner of the file view, and you should be able to add those minus signs and create a pull request for me to include them.

**Comment by elkins0** [comment]
This issue has been resolved by #87.
